cds/modules/fixtures/data/keywords.json

```diff
@@ -0,0 +1,7 @@
+[
+  {"key": "1", "value": "keyword 1"},
+  {"key": "2", "value": "keyword 2"},
+  {"key": "3", "value": "keyword 3"},
```

Comment:
are we changing the structure of Drupal's api on the task? AFAIR the structure is like
```json
[
  {
    "name": "13 TeV"
  }
]
```